src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/BindableTypeProvidersGenerationTask.cs

```diff
@@ -198,18 +177,11 @@ private bool IsValidProvider(INamedTypeSymbol type)
 			private void GenerateProviderTable(IEnumerable<INamedTypeSymbol> q, IndentedStringBuilder writer)
 			{
 				writer.AppendLineInvariant("[System.Runtime.CompilerServices.CompilerGeneratedAttribute]");
-				writer.AppendLineInvariant("[System.Diagnostics.CodeAnalysis.SuppressMessage(\"Microsoft.Maintainability\", \"CA1502:AvoidExcessiveComplexity\", Justification=\"Must be ignored even if generated code is checked.\")]");
-				writer.AppendLineInvariant("[System.Diagnostics.CodeAnalysis.SuppressMessage(\"Microsoft.Maintainability\", \"CA1506:AvoidExcessiveClassCoupling\", Justification = \"Must be ignored even if generated code is checked.\")]");
```

Comment:
I'm not sure why these originally existed. Both CA1502 and CA1506 don't trigger on auto-generated code.

Reply:
I'll revert that. But I'm under the impression that there shouldn't be any functional changes with these lines (if there is, it would certainly be a roslyn bug).